aptos-move/framework/aptos-stdlib/doc/overview.md

```diff
@@ -28,6 +28,7 @@ This is the reference documentation of the Aptos standard library.
 -  [`0x1::ristretto255`](ristretto255.md#0x1_ristretto255)
 -  [`0x1::secp256k1`](secp256k1.md#0x1_secp256k1)
 -  [`0x1::simple_map`](simple_map.md#0x1_simple_map)
+-  [`0x1::smart_tree`](smart_tree.md#0x1_smart_tree)
```

Comment:
looks like this got dirtied somehow from a previous commit.

Reply:
Removed.